Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -554,6 +554,13 @@ void ProgramShaderCache::CreateHeader()
 	break;
 	}
 
+	const char* earlyz_string = g_ogl_config.bSupportsEarlyFragmentTests ?
+	            "#extension GL_ARB_shader_image_load_store : enable\n"
+	            "#define FORCE_EARLY_Z layout(early_fragment_tests) in\n" :
+	            // Fallback method:
```

Comment:
This comment leaves important questions open:

* Fallback for what?
* What's the general idea behind this fallback?
* Why do we need both code paths?
* Is there any disadvantage in falling back?